src/extractors/custom/www.ossnews.jp/index.js

```diff
@@ -0,0 +1,27 @@
+export const WwwOssnewsJpExtractor = {
+  domain: 'www.ossnews.jp',
+
+  title: {
+    selectors: ['#alpha-block div h1.hxnewstitle'],
```

Comment:
I think we could do without the `div` in between, was there any specific case where you needed to add this in order for the selector to match?

Reply:
Thank you. `div` was not necessary.